src/cli/moz-cache.js

```diff
@@ -0,0 +1,217 @@
+import { Scheduler } from 'taskcluster-client';
+import slugid from 'slugid';
+
+/*
+ * This script is meant to be invoked as:
+ *
+ *    tc-vcs moz-cache
+ *
+ * This prepopulates a number of caches used by Mozilla CI.
+ *
+ */
+
+export default async function main(config, argv) {
+
+    var clones = [
```

Comment:
I think this is ok for now, but maybe in a future iteration we move this out into some config that could be loaded, and also allow overriding with command arguments, like if I wanted to create a cache for only one or two repos.